src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -117,15 +120,12 @@ Status OpenPrefixedKey(RedisModuleKey **out, RedisModuleCtx *ctx,
 
 /// Open the key used to store the channels that should be published to when an
 /// update happens at the given keyname.
-Status OpenBroadcastKey(RedisModuleKey **out, RedisModuleCtx *ctx,
-                        RedisModuleString *pubsub_channel_str, RedisModuleString *keyname,
-                        int mode) {
+std::string GetBroadcastKey(RedisModuleCtx *ctx, RedisModuleString *pubsub_channel_str,
+                            RedisModuleString *keyname) {
   RedisModuleString *channel;
-  RAY_RETURN_NOT_OK(FormatPubsubChannel(&channel, ctx, pubsub_channel_str, keyname));
+  RAY_CHECK_OK(FormatPubsubChannel(&channel, ctx, pubsub_channel_str, keyname));
```

Comment:
@guoyuhong removed all `RAY_CHECK` calls in https://github.com/ray-project/ray/pull/3855. So let's keep it that way.

Reply:
good idea, done